.eslintrc

```diff
@@ -43,7 +43,7 @@
       "never"
     ],
     "one-var": [
-      2,
+      0,
```

Comment:
Makes sense, great idea turning this off. I testes everything and it looks great and works so from me it's approved 😃